docs/rfcs/0008-websocket-responses-api.md

```
@@ -0,0 +1,657 @@
+# RFC 0008: Implement the WebSocket Responses API in axinite with
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)